### PR TITLE
kitti evaluation of range-merged result

### PR DESCRIPTION
**Pull request by leonzz**
Call kitti evaluation on saved results from pickle file.

Sample outputs:
```
python tools/merge_predictions/evaluate_kitti_range.py

Model trained with full range data:
Easy: 88.08   Moderate: 79.17   Hard: 76.29
Merged result from models handling separate ranges:
- Merge all:
Easy: 87.52   Moderate: 76.79   Hard: 73.70
- Merge with range cutoff at 34.0:
Easy: 87.96   Moderate: 78.56   Hard: 75.87
```

### CODE REVIEW

**Comment by nazMahmoud** [comment]
[1]- We would like to evaluate the performance of the model trained on full range [0-70m], on short range [0-35] and long range [30-70] so that we can compare them to the results of the models trained on short range and long range.

[2]- For our the second method, which reweights the loss function and trains on the entire dataset, implementing [1], would help us quantify performance on short versus long range. 

[3]- If it is possible, could you allow us to run evaluation from command line by passing the pickle file name containing the detections and the range of evaluation [0-35], [35-70], or full range [0-70]

**Comment by leonzz** [comment]
added another commit to use flags for file names, now:

- for the merge result: `python tools/merge_predictions/evaluate_kitti_range.py --short_range_result=tools/merge_predictions/result_short.pkl --long_range_result=tools/merge_predictions/result_long.pkl --merge_range_cutoff=34.0`

- for the full range result (or the model trained on modified loss function): `python tools/merge_predictions/evaluate_kitti_range.py --full_range_result tools/merge_predictions/result_fr.pkl`
